about.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone</title>
+    <link rel="stylesheet" href="about.css">
+    <script
+      src="https://kit.fontawesome.com/5d13e94c8b.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body>
+    <header class="header">
+        <div class="header-div">
+            <div>
+            <ul class="icons-ul">                
+                <li><img class="head-icon" src="./images/faceb.png" alt="facebookicon"></li>
+                <li><img class="head-icon" src="./images/twtr.png" alt="twitter-icon"></li>
+                <li class="link">Other language</li>
+                <li class="link">my page</li>
+                <li class="link">Logout</li>
+            </ul>
+            </div>
+            <div class="bars">
+                 <nav>
+                    <div class="span-"><img class="inner-backg" src="./images/download.jpg" alt=""><p class="logo">LIVING FAITH CHURCH</p> </div>
+
+                    <ul class="nav-ul">
+                        <li><a href="main.html">Home</a></li>
+                        <li>Program</li>
+                        <li>Join</li>
+                        <li>Sponsor</li>
+                        <li>News</li>
+                        <li><h3 class="winners">Winners Chapel</h3></li>
+                    </ul>
+                </nav>
+            </div>  
+        </div>
```

Comment:
- [ ] I am not able to navigate between pages because you did not add links, you have `about` and `home` pages but they are not linked to each other please address that

Reply:
done